connectors/src/connectors/microsoft/temporal/spreadsheets.ts

```diff
@@ -245,3 +238,33 @@ export async function syncSpreadSheet({
 
   return { isSupported: true };
 }
+
+export function isMicrosoftSpreadsheet(file: MicrosoftNodeResource): boolean {
+  return (
+    file.mimeType ===
+      "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" ||
+    file.mimeType === "application/vnd.ms-excel"
+  );
+}
```

Comment:
Are we sure we want to treat CSV and spreadsheet the same way? In your case this function makes total sense but I am afraid this might be reused wrongly in other places because of the name.

Reply:
Ok, will kill it :+1: